### DISCUSSION TITLE
One Pico W board will connect to wifi, one other one will not

### DISCUSSION THREAD

**Discussion by MABeatty1978** [micropython/micropython#10416] [RP2040 / Pico]
Hello,
I have two Pico W boards.  They are both running identical micropython firmware.  They are both sitting right next to each other using the same power supply.  One of the boards will connect to my Wifi (Google WiFi), one will not.  I'm using the same basic program provided in the [Pico W Datasheet](https://datasheets.raspberrypi.com/picow/connecting-to-the-internet-with-pico-w.pdf) on both of them and I'm getting a -1 return code from my wlan.connect(ssid, password) call after about 3 seconds of trying to connect on the bad one.  

I've created a hotspot on my phone and the bad one will connect to that just fine.  I've verified that the mac address are different.  I've tried to disable power saving mode.  Unfortunatley, Google doesn't give me much in the way of logs to look at for my router, so I don't think I can see what's going on in there.    

Any help would be appreciated...I've been beating my head against a wall on this for a couple of days.

EDIT: 
I just noticed something which leads back to there being a defective chip.
Durring my WLAN.scan(), I'm noticing that on the one that is not working, all of the RSSI values are coming back with really low numbers, -81, -94, -88, -92

Those same RSSI values are showing
-48, -55, -48, -43 respectively

**Comment by MABeatty1978** [comment]
I just noticed something which leads back to there being a defective chip.
Durring my WLAN.scan(), I'm noticing that on the one that is not working, all of the RSSI values are coming back with really low numbers, -81, -94, -88, -92

Those same RSSI values are showing
-48, -55, -48, -43 respectively

**Reply by robert-hh** [reply]
So it looks like a problem with the antenna-connection. Hard to tell given that the components are so tiny.
